draft-ietf-quic-transport.md

```diff
@@ -2288,15 +2298,11 @@ local address.  Failure of path validation simply means that the new path is not
 usable for this connection.  Failure to validate a path does not cause the
 connection to end unless there are no valid alternative paths available.
 
-An endpoint uses a new connection ID for probes sent from a new local address;
-see {{migration-linkability}} for further discussion. An endpoint that uses
-a new local address needs to ensure that at least one new connection ID is
-available at the peer. That can be achieved by including a NEW_CONNECTION_ID
-frame in the probe.
-
-Receiving a PATH_CHALLENGE frame from a peer indicates that the peer is probing
-for reachability on a path. An endpoint sends a PATH_RESPONSE frame in response,
-as per {{migrate-validate}}.
+An endpoint uses a new connection ID for probes sent from a new local address;	
```

Comment:
I think this would be clearer if we say that the "endpoint" we're talking about here is a client.

Reply:
This is the path validation section, not the connection migration section, FYI.  I think the intent is that path validation not be client or server specific?